.eslintrc.js

```diff
@@ -1,6 +1,21 @@
 /** @type {import('eslint').Linter.Config} */
 const config = {
-  extends: ['ackama']
+  parserOptions: {
+    sourceType: 'module'
+  },
+  env: {
+    browser: true,
+    commonjs: true,
+    node: true
+  },
+  extends: ['ackama'],
+  ignorePatterns: [
+    '/node_modules/',
+    '/vendor/',
+    '/themes/watea/',
```

Comment:
```suggestion
  ignorePatterns: [
    '/themes/watea/',
```

These are already included by our `ackama` config.